### ISSUE TITLE
Unexpected exception

### ISSUE THREAD

**Issue by qualishovt** [apache/netbeans#5650]
### Apache NetBeans version

Apache NetBeans 17

### What happened

```
java.lang.NullPointerException: Cannot invoke "java.util.Collection.toArray()" because "c" is null
	at java.base/java.util.ArrayList.<init>(ArrayList.java:181)
	at org.netbeans.modules.web.indent.api.support.IndenterFormattingContext.getAndClearChanges(IndenterFormattingContext.java:85)
	at org.netbeans.modules.web.indent.api.support.AbstractIndenter.reindent(AbstractIndenter.java:203)
	at org.netbeans.modules.html.editor.indent.HtmlIndentTask.reindent(HtmlIndentTask.java:49)
	at de.funfried.netbeans.plugins.external.formatter.ui.editor.ExternalFormatterIndentTaskFactory.formatWithNetBeansIndenter(ExternalFormatterIndentTaskFactory.java:151)
	at de.funfried.netbeans.plugins.external.formatter.ui.editor.ExternalFormatterIndentTaskFactory$2.reindent(ExternalFormatterIndentTaskFactory.java:96)
	at org.netbeans.modules.editor.indent.TaskHandler$MimeItem.runTask(TaskHandler.java:548)
	at org.netbeans.modules.editor.indent.TaskHandler.runTasks(TaskHandler.java:309)
	at org.netbeans.modules.editor.indent.IndentImpl.reformat(IndentImpl.java:349)
	at org.netbeans.modules.editor.indent.api.Reformat.reformat(Reformat.java:129)
	at org.netbeans.editor.ActionFactory$FormatAction$1$1.run(ActionFactory.java:1862)
	at org.netbeans.editor.GuardedDocument.runAtomicAsUser(GuardedDocument.java:335)
	at org.netbeans.editor.ActionFactory$FormatAction$1.run(ActionFactory.java:1841)
	at org.netbeans.modules.progress.ui.RunOffEDTImpl$1.run(RunOffEDTImpl.java:146)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1418)
	at org.netbeans.modules.openide.util.GlobalLookup.execute(GlobalLookup.java:45)
	at org.openide.util.lookup.Lookups.executeWith(Lookups.java:287)
[catch] at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:2033)
```

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Windows 11

### JDK

18

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

No

**Comment by OndroMih** [comment]
This happens for me in Netbeans 18, Ubuntu Linux, Java 17. I reproduced it with the following steps:

* installed the External Code Formatters plugin. No need to configure anything in Options.
* On any project (I tried a Java maven web project), select the project root directory, click Source in the top menu, then Format, then OK
* Formatting starts but stops almost immediately, with this error in the Notifications window

When I uninstall the External Code Formatters plugin, all works.

Maybe it's better to raise this in the plugin project? https://github.com/funfried/externalcodeformatter_for_netbeans
